src/plugins/media-analysis/media-analysis-utils.js

```diff
@@ -54,6 +65,65 @@ export function createDefaultContent(t) {
   };
 }
 
+export function exportChaptersToCsv(chapters) {
+  return [
+    'startPosition,title,color,text',
+    ...chapters.map(chapter => [
+      chapter.startPosition.toString(),
+      `"${chapter.title.replace(/"/g, '""')}"`,
+      `"${chapter.color.replace(/"/g, '""')}"`,
+      `"${chapter.text.replace(/"/g, '""')}"`
```

Comment:
Maybe it's worth extracting an "escapeText" method here.

Reply:
True. Actually it was pretty straight forward to extract generic `(to|from)Csv` functions, so I created a `csv-utils` module.

Reply:
Ah, great! 👍